packages/components/src/widgets/LambdaWidget/index.tsx

```diff
@@ -18,6 +21,14 @@ widgetRegistry.register("Lambda", {
   },
   Chart: (value, settings) => {
     const environment = value.context.project.getEnvironment();
+    //It's kind of awkward that the documentation isn't connected to the function itself, but that's a greater effort.
```

Comment:
Yeah. We could split `SqLambda` into two subclasses and expose all documentation-related fields on it with methods, that's pretty straightforward. Then we won't need a separate `getFunctionDocumentation`.

Reply:
Could do some this in ``BuildInLambda`` directly, maybe a bit later on? That would be more convenient.